eras/babbage/impl/src/Cardano/Ledger/Babbage/Rules/Utxo.hs

```diff
@@ -266,7 +266,14 @@ validateTotalCollateral pp txBody utxoCollateral =
 -- non-ada collateral, but only if the same amount of the same multi-asset is
 -- present in the collateral return output.
 --
```

Comment:
We could remove the two lines above (and half of the line above that, but github won't let me select it), since you explained in a much better way in the new comment. 
```suggestion

```
But it won't hurt to have it explained twice, anyway!

Reply:
Thanks for pointing it out. Updated the wording to reduce duplication